### DISCUSSION TITLE
How to compare the performance between fastdds and zmq or fastdds and opendds?

### DISCUSSION THREAD

**Discussion by wddpfx** [eProsima/Fast-DDS#4495] [General]
I am a new hand. In https://www.eprosima.com/index.php, The result is displayed, But I want to do it myself, What tools or methods i can use?
![image_23](https://github.com/eProsima/Fast-DDS/assets/19266720/8502c6b2-665a-4ee3-a25e-1b6bf69e3de2)

**Comment by JesusPoderoso** [comment]
Hi @wddpfx, in each of those links it has been described the test description, specification, methodology, and configuration.
